eureka-core/src/main/java/com/netflix/eureka/aws/AwsAsgUtil.java

```diff
@@ -371,6 +371,11 @@ private TimerTask getASGUpdateTask() {
             @Override
             public void run() {
                 try {
+                    if (!serverConfig.shouldUseAwsAsgApi()) {
```

Comment:
You'll also need to handle the async refresh on line 70.

Reply:
Resolved